core/src/test/java/com/graphhopper/routing/util/RacingBikeFlagEncoderTest.java

```diff
@@ -206,75 +208,92 @@ public void testHandleWayTagsInfluencedByRelation() {
     }
 
     @Test
-    public void testAvoidanceOfHighMaxSpeed() {
+    public void testPriority_avoidanceOfHighMaxSpeed() {
+        // here we test the priority that would be calculated if the way was accessible (even when it is not)
+        // therefore we need a modified encoder that always yields access=WAY
+        BikeCommonFlagEncoder encoder = new RacingBikeFlagEncoder(new PMap("block_fords=true")) {
+            @Override
+            public EncodingManager.Access getAccess(ReaderWay way) {
+                return WAY;
+            }
+        };
```

Comment:
This is the same problem I described in the other comment: We used to be able to test the priority by ignoring access by just passing in `Access.WAY`. Since this is no longer possible I worked around this by using a modified encoder here. The much better approach would of course be testing the priority calculation in isolation, but this is not possible yet, because `handleWayTags` does a bit of everything.